README.md

```diff
@@ -70,6 +71,21 @@ parser.end();
 
 var parsedManifest = parser.manifest;
 ```
+### Constructor Options
+
+The constructor optinally takes an options object with two properties. These are needed when using `#EXT-X-DEFINE` for variable replacement.
+
+```js
+var parser = new m3u8Parser.Parser({
```

Comment:
this seems reasonable to me. The only thing to consider is whether the options property for definitions should match the value provided by the parser.
So that you could pass the output of the parser from the main playlist directly in to the parser for the media playlist.

Reply:
The ideas was that `mainDefinitions: main.defintions` would be added to the object [here](https://github.com/videojs/http-streaming/blob/d6851ccee1cc45173f7bbbfb6b7944293d2cd19a/src/playlist-loader.js#L568-L571), and `parseManifest_` would pass that on.